config/blacklight.yml

```diff
@@ -6,8 +6,8 @@ test: &test
   url: <%= ENV.fetch('SCHOLARSARCHIVE_SOLR_URL', 'http://localhost:8985') %>/solr/hydra-test
 production:
   adapter: solr
-  url: <%= ENV.fetch('SCHOLARSARCHIVE_SOLR_URL', 'http://localhost:8080/solr/hydra-production') %>
+  url: <%= ENV.fetch('SCHOLARSARCHIVE_SOLR_URL', 'http://localhost:8080/solr/production') %>
```

Comment:
Is this production safe?

Reply:
yes, SCHOLARSARCHIVE_SOLR_URL is set for prod, and that value doesn't match production anyways. I did update the file to use the same SOLR core name at least (hydra-production -> hydra-prod)